client/app/components/queries/QueryEditor/AutoLimitCheckbox.jsx

```diff
@@ -0,0 +1,28 @@
+import React, { useCallback } from "react";
+import PropTypes from "prop-types";
+import "@/redash-font/style.less";
+import recordEvent from "@/services/recordEvent";
+import { Checkbox } from "antd";
+
+export default function AutoLimitCheckbox({ available, checked, onChange }) {
+  const handleClick = useCallback(() => {
+    recordEvent("checkbox_auto_limit", "screen", "query_editor", { state: !checked });
+    onChange(!checked);
+  }, [checked, onChange]);
+
+  return (
+    <Checkbox
+      className="query-editor-controls-checkbox m-l-5"
```

Comment:
```suggestion
      className="query-editor-controls-checkbox"
```

This is covered in the `margin: auto 5px;`